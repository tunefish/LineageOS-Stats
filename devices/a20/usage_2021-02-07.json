{
  "17.1": 2416,
  "none": 21
}