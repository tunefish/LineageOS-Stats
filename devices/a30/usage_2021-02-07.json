{
  "17.1": 28,
  "18.0": 620,
  "18.1": 163,
  "none": 4
}