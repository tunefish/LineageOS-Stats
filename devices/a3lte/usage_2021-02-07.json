{
  "15.1": 536,
  "16.0": 186,
  "17.1": 614
}