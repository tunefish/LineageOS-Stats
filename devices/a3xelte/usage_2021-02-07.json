{
  "15.1": 105,
  "16.0": 451,
  "17.1": 3295,
  "18.0": 3,
  "18.1": 11,
  "52.8": 1,
  "none": 2
}