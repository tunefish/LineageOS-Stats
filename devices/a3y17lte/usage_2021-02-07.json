{
  "14.1": 5420,
  "15.1": 333,
  "16.0": 965,
  "17.1": 913,
  "18.0": 1
}