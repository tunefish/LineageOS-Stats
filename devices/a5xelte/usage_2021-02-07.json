{
  "15.1": 53,
  "16.0": 514,
  "17.1": 3963,
  "18.0": 12,
  "18.1": 2,
  "none": 1
}