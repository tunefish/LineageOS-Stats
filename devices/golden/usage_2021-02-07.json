{
  "13.0": 991,
  "14.1": 3197,
  "15.1": 4,
  "17.1": 4,
  "none": 1
}